consumer/consumer.go

```diff
@@ -258,6 +277,41 @@ func organizationAllowed(consumer *KafkaConsumer, orgID types.OrgID) bool {
 	return orgWhitelisted
 }
 
+func (consumer *KafkaConsumer) trackPayload(reqID interface{}, timestamp time.Time, status string) {
+	reqIDString, ok := reqID.(string)
+	if !ok {
+		log.Warn().Msgf("request ID is not a string: %#v", reqID)
+		return
+	}
+
+	_, _, err := consumer.payloadTrackerProducer.ProduceMessage(producer.PayloadTrackerMessage{
+		Service:   "insights-results-aggregator",
```

Comment:
ok for PoC, might be better to have it configurable later

Reply:
Now configurable via `broker.service_name`.